### DISCUSSION TITLE
Thermodynamic Properties of CO2 - R.Span and W.Wagner

### DISCUSSION THREAD

**Discussion by andreubernadserra** [idaholab/moose#17104] [Q&A Modules: General] [answered]
The values for` _a4` and `_gamma3` in header file `CO2FluidProperties.h` differ slightly from those reported in the original reference:

- The last value of the ` _a4` array is 3.000 (instead of the implemented 3.500) according to Table 31 of the cited reference.
- The last value of the  `_gamma3` array is 1.22 (instead of the implemented 1.25) according to mentioned table.

Were these values modified on purpose? Nevertheless, this should have no impact on the calculations.

Thanks in advance and have a nice day!

**Comment by WilkAndy** [comment]
@cpgr ?

**Comment by cpgr** [comment]
I would assume these are simply typos I made when I was writing it! It probably (hopefully) won't affect the calculations appreciably as you say, but should be fixed up. Thanks for spotting it!

**Comment by cpgr** [comment, marked as answer]
I've fixed this in #17119. Thanks again for spotting this!

**Reply by andreubernadserra** [reply]
Thank you for the prompt response. Greetings from Barcelona!
